AndroidAnnotations/androidannotations-otto/otto/src/main/java/org/androidannotations/otto/handler/AbstractOttoHandler.java

```diff
@@ -64,6 +64,10 @@ public void validate(Element element, ElementValidation valid) {
 
 	@Override
 	public void process(Element element, EComponentHolder holder) throws Exception {
+		if (!annotationHelper.enclosingElementHasEnhancedComponentAnnotation(element)) {
+			// do nothing when otto annotations are used in non-enhanced classes
+			return;
+		}
```

Comment:
Just to be clear, i am talking about this. https://github.com/dodgex/androidannotations/blob/1657_ignore_otto_annotions_for_not_enhanced_classes/AndroidAnnotations/androidannotations-otto/otto/src/main/java/org/androidannotations/otto/handler/AbstractOttoHandler.java#L48